### DISCUSSION TITLE
Can Appimage call host-side program?

### DISCUSSION THREAD

**Discussion by qlibp** [AppImage/AppImageKit#1377] [Q&A]
I use an nixvim appimage, containing a pylsp which tries to access host-side venv’s python. But it raises fuse error. Not sure if this is the recursive fuse error?

**Comment by probonopd** [comment]
Please post the command that fails, and the error. Thanks!

**Reply by qlibp** [reply]
I think the root cause is using the nix bundled appimage inside a nix devshell, which have same /nix/store path, leading to a conflict path between host & appimage file system

**Comment by TheAssassin** [comment]
The answer is "yes", in general. Your definition of "host environment" may vary, though. nix sets up separate contexts and isolates environments. Any kind of restriction there is not introduced by AppImage as a technology.
